Godeps/_workspace/src/k8s.io/kubernetes/pkg/client/unversioned/services.go

```diff
@@ -36,7 +37,7 @@ type ServiceInterface interface {
 	Update(srv *api.Service) (*api.Service, error)
 	Delete(name string) error
 	Watch(label labels.Selector, field fields.Selector, resourceVersion string) (watch.Interface, error)
-	ProxyGet(name, path string, params map[string]string) ResponseWrapper
+	ProxyGet(scheme, name, port, path string, params map[string]string) ResponseWrapper
```

Comment:
If we're proxying a URL, I'd use a URL.  Host +port is a detail of config,
people using proxy get today are assuming port = scheme anyway.

On Nov 7, 2015, at 9:55 AM, Jordan Liggitt <notifications@github.com> wrote:

In
Godeps/_workspace/src/k8s.io/kubernetes/pkg/client/unversioned/services.go
<https://github.com/openshift/origin/pull/5763#discussion_r44213668>:

> @@ -36,7 +37,7 @@ type ServiceInterface interface {
>  	Update(srv *api.Service) (*api.Service, error)
>  	Delete(name string) error
>  	Watch(label labels.Selector, field fields.Selector, resourceVersion string) (watch.Interface, error)
> -	ProxyGet(name, path string, params map[string]string) ResponseWrapper
> +	ProxyGet(scheme, name, port, path string, params map[string]string) ResponseWrapper

where host is service:port?

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/5763/files#r44213668>.